package.json

```diff
@@ -1,10 +1,11 @@
 {
-  "name": "macOSNotifJS",
-  "version": "0.0.5",
+  "name": "macosnotif",
+  "version": "0.0.6",
   "description": "A simple Javascript plugin to create simulated macOS notifications on your website.",
-  "main": "src/macOSNotif.js",
+  "browser": "src/macOSNotif.js",
```

Comment:
I wonder if actually, `main` should remain as `src/macOSNotif.js` and then `browser` should point to `dist/macOSNotif.min.js`?